SGTabbedPager/SGTabbedPager.cs

```diff
@@ -174,9 +174,26 @@ public override void LoadView()
         public override void ViewWillAppear(bool animated)
         {
             base.ViewWillAppear(animated);
+
+            TitleScrollView.Delegate = this;
+            ContentScrollView.Delegate = this;
+
             ReloadData();
         }
 
+        public override void ViewWillDisappear(bool animated)
+        {
+            base.ViewWillDisappear(animated);
+
+            foreach (var btn in _tabButtons)
+            {
+                btn.RemoveTarget(ReceivedButtonTab, UIControlEvent.TouchUpInside);
```

Comment:
They are re-added from scratch in GenerateTabs that gets called in ReloadData that in turn gets called in ViewWillAppear.